torch/onnx/symbolic.py

```diff
@@ -1658,7 +1658,7 @@ def detach(g, input):
     return input
 
 
-def contiguous(g, input):
+def contiguous(g, input, memory_format=None):
```

Comment:
This looks wrong; if `memory_format` is not None we should raise an error as we cannot properly ONNX export it